cmd/src-fingerprint/main.go

```diff
@@ -51,8 +52,12 @@ func getExporter(exporterStr string, output io.Writer) (exporter.Exporter, error
 	switch exporterStr {
 	case "json":
 		return exporter.NewJSONExporter(output), nil
+	case "gzip-json":
+		return exporter.NewGzipJSONExporter(output), nil
 	case "jsonl":
 		return exporter.NewJSONLExporter(output), nil
+	case "gzip-jsonl":
+		return exporter.NewGzipJSONLExporter(output), nil
```

Comment:
Note : See L198/210, we never use the error message hereunder. We may want to change L198/210 to print out this error message in addition to the command help menu. WDYT ?

Reply:
I've added `log.Errorln` before the `cli.ShowAppHelpAndExit` calls.